DSharpPlus/Clients/SingleShardOrchestrator.cs

```diff
@@ -36,8 +36,15 @@ PayloadDecompressor decompressor
     public bool AllShardsConnected => this.gatewayClient.IsConnected;
 
     /// <inheritdoc/>
-    public ValueTask BroadcastOutboundEventAsync(byte[] payload) => SendOutboundEventAsync(payload, 0);
+    public async ValueTask BroadcastOutboundEventAsync(byte[] payload)
+    {
+        if (!this.AllShardsConnected)
+        {
+            throw new InvalidOperationException("Broadcast is only possible when the shard is connected");
```

Comment:
s/shard/client

Reply:
I have no idea what you want to tell me

Reply:
I would replace shard with client in the case of a single shard, but that's just personal opinion

Reply:
Yeah i thought about it, but we introduce one sharded users to the term shards in other places and i think its a concept ppl should know

Reply:
Also this exception is very unlikely to be thrown when using the one sharded client